cdp_scrapers/instances/seattle.py

```diff
@@ -314,7 +311,7 @@ def get_video_page_urls(
         for link in soup.find(
             "div",
             class_="paginationContainer",
-        ).find_all(
+            ).find_all(
```

Comment:
I feel like linting would fail this / `black` would format this (just guessing).